exporter/exporterhelper/internal/persistent_storage.go

```diff
@@ -360,6 +360,8 @@ func (pcs *persistentContiguousStorage) itemDispatchingStart(ctx context.Context
 		pcs.logger.Debug("Failed updating currently dispatched items",
 			zap.String(zapQueueNameKey, pcs.queueName), zap.Error(err))
 	}
+
+	recordCurrentlyDispatchedBatches(ctx, len(pcs.currentlyDispatchedItems), pcs.queueName)
```

Comment:
Do we only record these new metrics for persistent queue or also for in-memory queue when persistent queue is disabled?
I think they are useful regardless of whether persistence is enabled or no.

Reply:
Yeah, I was wondering about doing that for in-memory queue as well though it seems it's using the old API and metrics would need to be pretty much redone there